statistics/handle/cache/internal/lru/bench_test.go

```diff
@@ -0,0 +1,80 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package lru
+
+import (
+	"sync"
+	"testing"
+
+	"github.com/pingcap/tidb/statistics/handle/cache/internal/testutil"
+	"github.com/pingcap/tidb/util/benchdaily"
+)
+
+const defaultSize int64 = 1000
+
+func BenchmarkLruPut(b *testing.B) {
```

Comment:
It seems like `BenchmarkLRUXXX` is duplicated with `BenchmarkMapXXX`.
Why not we implement these benchmark methods on `StastCacheInner` interface, then we can remove some duplicated code, for example:
```
func benchmarkStatsCacheInner(b, c) {
...
}

func Benchmark(b) {
    for c := []Cache{LRU, Map} {
        benchmarkStatsCacheInner(b, c)
    }
}
```

Reply:
Done